src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -182,60 +183,18 @@ private static void AddAnalyzerConfigDocumentWithOptions(TestWorkspace workspace
             Assert.True(applied);
             return;
 
-            static string GenerateAnalyzerConfigText(IDictionary<OptionKey, object> options)
+            string GenerateAnalyzerConfigText(TestParametersOptions options)
             {
                 var textBuilder = new StringBuilder();
 
                 // Add an auto-generated header at the top so we can skip this file in expected baseline validation.
                 textBuilder.AppendLine(AutoGeneratedAnalyzerConfigHeader);
                 textBuilder.AppendLine();
-
-                foreach (var (optionKey, value) in options)
```

Comment:
We now re-use the code that Sam added in `CodeFixVerifier` that serializes editorconfig options to editorconfig string for tests based on new testing framework.